doc/api/cli.md

```diff
@@ -33,7 +33,8 @@ If a file is found, its path will be passed to the
 
 * The program was started with a command-line flag that forces the entry
   point to be loaded with ECMAScript module loader, such as `--import`.
-* The file has an `.mjs` extension.
+* The file has an `.mjs`, `.mts` (with `--experimental-strip-types`) or `.wasm`
```

Comment:
isn't this unflagged now?

Reply:
I'm just going by what's implemented in run main - https://github.com/nodejs/node/pull/57610/files#diff-9cdd961b23c4d25cc188f6b0beb3bf0228e3696a5f830cef198b1673b547bed3.

Reply:
Actually you're right thanks, it seems this is a deprecation path. Removed.

Reply:
what i mean is, `mts` just works out of the box now, without any flags, so it should be added, just without the parenthetical.

Reply:
even for mts? very weird :-) will resolve then, thanks!

Reply:
Not sure where you got that information, .mts are supported by both cjs and esm loader, exactly how the cjs loader supports .mjs.

Reply:
Right, but this documentation section is about which loader is being used, and the CJS loader is being used for .cjs. We should probably overhaul this documentation section though now that there is loader equivalence, although it is worth noting `.wasm` imports aren't supported yet in the CJS loader.